src/controllers/update/relevant.period.owned.land.filter.controller.ts

```diff
@@ -27,11 +30,13 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 
 export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
-    logger.debugRequest(req, `POST ${config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE}`);
-    const ownedLandKey = req.body[OwnedLandKey];
+    beforeRender(req, res, postMethod);
+    const ownedLandKey = req.body[OwnedLandKey] ?? '';
 
     if (ownedLandKey === '1') {
       return res.redirect(config.RELEVANT_PERIOD_INTERRUPT_URL);
+    } else if (ownedLandKey !== '0' && ownedLandKey === '') {
```

Comment:
In theory this should be handled by the relevant.period.owned.land.validation right? so we might not have to check for this error in the controller

Reply:
Yes @ConnorOS-ch this could be resolved now, as you know, relevant.period.owned.land.validation.ts should have been handling this validation, instead of double checking for this in the controller. Removed now and also see your own merged PR https://github.com/companieshouse/overseas-entities-web/pull/1432 Thanks.

Reply:
Nice, yeah I notice that that bug in navigation was stoping crashing the validation middleware, glad we could get that fixed 👍